cylc/flow/scripts/cylc.py

```diff
@@ -283,7 +283,8 @@ def get_version(long=False):
     'gscan':
         'cylc gscan has been removed, use the web UI',
     'insert':
-        'inserting tasks is now done automatically',
+        'Insertion is now automatic: Use `cylc set` or `cylc trigger`'
+        ' to re-run old tasks.',
```

Comment:
Insert can also be used to run future tasks:

```suggestion
        'Insertion is no longer required, `cylc set` and `cylc trigger`'
        ' will insert tasks automatically.',
```